styles/main.css

```diff
@@ -1,4 +1,83 @@
 body {
   margin: 0;
+  background: #f7f7f7;
   padding: 0;
 }
+.container {
+  padding: 4rem 8.3%;
+}
+.home-top-grid {
+  justify-content: space-between;
+  gap: 1rem;
+}
+.home-top-grid-item {
+  height: 8.75rem;
+  width: 100%;
+  background-image: url('/assets/images/blazer.png');
+  background-repeat: no-repeat, repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.center-overlay {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 1rem;
+  line-height: 1rem;
+  padding: 0.313rem 0;
+  text-align: center;
+  left: 0;
+  right: 0;
+  background-color: #fff;
+}
+
+.home-middle-section {
+  background: #c4c4c4;
```

Comment:
Code looks good.Few suggestions: Use color variables in root folder so it is easir to reuse